src/abstractions/software/assembler/writter/index.js

```diff
@@ -0,0 +1,15 @@
+class Writter {
+  constructor () {
+    this.machineCode = ''
+  }
+
+  write (instruction) {
+    this.machineCode += instruction + '\n'
+  }
+
+  get () {
```

Comment:
simply naming it as `get` might be a little confusing
```suggestion
  getResult () {
```